src/ontoweaver/base.py

```diff
@@ -508,6 +504,17 @@ def __repr__(self):
 
         return f"<Transformer:{type(self).__name__}({params}) {','.join(columns)}{link}>"
 
+    def create(self, item):
+
+        try:
+            res = str(item)
+            if self.output_validator(pd.DataFrame([res], columns=["cell_value"])):
+                yield res
+        except pa.errors.SchemaErrors as error:
+            msg = f"Transformer {self.__repr__()} did not produce valid data {error}."
+            logging.error(msg)
```

Comment:
Use the ErrorManager interface for managing errors:
```suggestion
            self.error(msg, exception = exceptions.DataValidationError)
```